scripts/check_license.sh

```diff
@@ -18,7 +18,7 @@ check_file() {
         SUFFIX1='*.h'
     else
         SUFFIX0='*.sql'
-        SUFFIX1='*.sql'
+        SUFFIX1='*.sql.in'
     fi
 
     find $2 -type f \( -name "${SUFFIX0}" -or -name "${SUFFIX1}" \) -and -not -path "${SRC_DIR}/sql/updates/*.sql" -and -not -path "${SRC_DIR}/test/sql/dump/*.sql" -and -not -path "${SRC_DIR}/src/chunk_adaptive.*" -print0 | xargs -0 -n1 $(dirname ${0})/check_file_license.sh ${1}
```

Comment:
Even though this works, it is probably easier to use grep to filter files for this case (something like `find ... | grep ... | sh check_license.sh`) . That way you can build a quite complex regex. Nothing that needs to be changed in this commit though.